bytecomp/bytegen.ml

```diff
@@ -576,7 +576,7 @@ let rec comp_expr env exp sz cont =
         in
         comp_init env sz decl_size
       end
-  | Lprim((Pidentity | Popaque), [arg]) ->
+  | Lprim((Pidentity | Popaque | Pbytes_to_string | Pbytes_of_string), [arg]) ->
```

Comment:
those two primitives are used in `Pervasives.unsafe_output`, `Pervasives.unsafe_output_string`, `String.unsafe_blit`, `Bytes.unsafe_blit`
The changes are minimal